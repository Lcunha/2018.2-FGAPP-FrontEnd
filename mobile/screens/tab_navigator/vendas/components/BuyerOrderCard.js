import React, { Component } from 'react';
import { Image, TouchableHighlight } from 'react-native';
import { Card, CardItem, Text, Left, Right, View } from 'native-base';
import styles from './styles';

class BuyerOrderCard extends Component {
  render() {
    return (
      <View style={{padding:10}}>
        <TouchableHighlight onPress={this.props.onPress}>
          <View style={styles.buyer_order_main}>
            <View style={{flexDirection: 'column', width: '65%',}}>
              <Text style={styles.order_title}>{this.props.orderName}</Text>
              <Text style={styles.order_subtitle}>{this.props.orderQuantity}</Text>
            </View>
            <View style={{flexDirection: 'column', width: '35%',}}>
              <Text style={styles.order_price}>R$ {this.props.orderPrice}</Text>
            </View>
            <View style={{height: 10}} />
          </View>
        </TouchableHighlight>
      </View>
    );
  }
}
export default BuyerOrderCard;
