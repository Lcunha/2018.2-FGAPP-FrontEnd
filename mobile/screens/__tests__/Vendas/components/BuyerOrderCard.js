import React from 'react';
import BuyerOrderCard from '../../../tab_navigator/vendas/components/BuyerOrderCard';
import Adapter from 'enzyme-adapter-react-16';
import {shallow} from 'enzyme';
import Enzyme from 'enzyme';
import renderer from 'react-test-renderer';

Enzyme.configure({adapter: new Adapter()});

it('renders correctly', () => {
  const tree = renderer.create(<BuyerOrderCard />).toJSON();
  expect(tree).toMatchSnapshot();
});
