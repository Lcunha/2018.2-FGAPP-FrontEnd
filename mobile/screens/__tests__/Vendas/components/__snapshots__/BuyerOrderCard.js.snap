// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {},
      Object {
        "padding": 10,
      },
    ]
  }
  virtual={undefined}
>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hasTVPreferredFocus={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={null}
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <View
      style={
        Array [
          Object {},
          Object {
            "backgroundColor": "#0EAC6F",
            "flex": 1,
            "flexDirection": "row",
            "paddingBottom": 20,
            "paddingLeft": 20,
            "paddingRight": 20,
            "paddingTop": 10,
          },
        ]
      }
      virtual={undefined}
    >
      <View
        style={
          Array [
            Object {},
            Object {
              "flexDirection": "column",
              "width": "65%",
            },
          ]
        }
        virtual={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#000",
                "fontFamily": "System",
                "fontSize": 16,
              },
              Object {
                "color": "white",
                "fontSize": 25,
              },
            ]
          }
          uppercase={false}
          virtual={undefined}
        />
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#000",
                "fontFamily": "System",
                "fontSize": 16,
              },
              Object {
                "color": "white",
                "fontSize": 20,
              },
            ]
          }
          uppercase={false}
          virtual={undefined}
        />
      </View>
      <View
        style={
          Array [
            Object {},
            Object {
              "flexDirection": "column",
              "width": "35%",
            },
          ]
        }
        virtual={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#000",
                "fontFamily": "System",
                "fontSize": 16,
              },
              Object {
                "color": "white",
                "fontSize": 25,
              },
            ]
          }
          uppercase={false}
          virtual={undefined}
        >
          R$ 
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#000",
                "fontFamily": "System",
                "fontSize": 16,
              },
              Object {
                "color": "white",
                "fontSize": 17,
              },
            ]
          }
          uppercase={false}
          virtual={undefined}
        >
           
          Cancelado
           
        </Text>
      </View>
      <View
        style={
          Array [
            Object {},
            Object {
              "height": 10,
            },
          ]
        }
        virtual={undefined}
      />
    </View>
  </View>
</View>
`;
